Source/Fuse.Platform/iOS/SystemUI.uno

```diff
@@ -320,5 +322,95 @@ namespace Fuse.Platform
 		{
 			IsTopFrameVisible = false;
 		}
+
+		public static ScreenOrientation DeviceOrientation
+		{
+			get
+			{
+				var orientation = GetCurrentScreenOrientation();
+				switch (orientation)
+				{
+					case 0:
+						return ScreenOrientation.Portrait;
+					case 1:
+						return ScreenOrientation.LandscapeLeft;
+					case 2:
+						return ScreenOrientation.LandscapeRight;
+					case 3:
+						return ScreenOrientation.PortraitUpsideDown;
+					default:
+						return ScreenOrientation.Default;
+				}
+			}
+			set
+			{
+				if (DeviceOrientation != value)
+				{
+					SetCurrentScreenOrientation(value);
+					if (DeviceOrientationChanged != null)
+						DeviceOrientationChanged(value);
+				}
+			}
+		}
+
+		[Foreign(Language.ObjC)]
+		static int GetCurrentScreenOrientation()
+		@{
+			if (@available(iOS 13, *))
+			{
+				 UIInterfaceOrientation mask = [[UIApplication sharedApplication].windows firstObject].windowScene.interfaceOrientation;
```

Comment:
Travis failed on this line complaining about `windowScene`, the property of `windowScene` on UIWindow is only available on iOS 13

Reply:
https://github.com/fuse-open/fuselibs/blob/master/Source/Fuse.Common/Internal/iOSSystemFont.uno demonstrates how to test for iOS13.

Compile-time check:
```
#if defined(__IPHONE_13_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_13_0

#endif
```

Run-time check:
```
if (Fuse.iOSDevice.OperatingSystemVersion.Major >= 13) 
{
}
```

Reply:
I thought using `@available(iOS 13, *)` is enough, well I try to replace with `#if` directive then